inc/wp-components/traits/trait-wp-widget-sidebar.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * WP_Widget_Sidebar trait.
+ *
+ * @package WP_Components
+ */
+
+namespace WP_Components;
+
+/**
+ * WP_Widget_Sidebar trait.
+ */
+trait WP_Widget_Sidebar {
+
+	/**
+	 * Mapping of widgets to their handlers.
+	 *
+	 * @var array
+	 */
+	public $sidebar_widget_mapping = [];
```

Comment:
Would this make sense as a static across all instances? I'm also considering filtering options. Since it's a trait it may be a bit tricky since there's no constructor, but let's chat about it.

Reply:
Updated to filter